### PR TITLE
✨File upload

### PR DESCRIPTION
**Pull request by Otavio-CB**

### CODE REVIEW

File: web-server/people.csv
Comment (pauloarantesmachado) [inline web-server/people.csv:1]: Is this file necessary?

File: web-server/people.xlsx
Comment (pauloarantesmachado) [inline web-server/people.xlsx:1]: Is this File necessary?

File: web-server/src/main/java/com/domrock/configurator/controller/SendJson.java
Comment (pauloarantesmachado) [inline web-server/src/main/java/com/domrock/configurator/controller/SendJson.java:1]: Suggestion: change @RequestMapping to  ("/convert") only, because route name "/api" api name not in context.

File: web-server/src/main/java/com/domrock/configurator/controller/SendJson.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.domrock.configurator.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
```
Comment (pauloarantesmachado) [inline web-server/src/main/java/com/domrock/configurator/controller/SendJson.java:6]: remove empty line

File: web-server/src/main/java/com/domrock/configurator/utils/ArrayListToJson.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.domrock.configurator.utils;
+
+import java.util.ArrayList;
+
```
Comment (pauloarantesmachado) [inline web-server/src/main/java/com/domrock/configurator/utils/ArrayListToJson.java:4]: remove empty line

File: web-server/src/main/java/com/domrock/configurator/utils/CsvConverter.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
+package com.domrock.configurator.utils;
+
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Cell;
+
```
Comment (pauloarantesmachado) [inline web-server/src/main/java/com/domrock/configurator/utils/CsvConverter.java:7]: remove empty line

File: web-server/src/main/java/com/domrock/configurator/utils/CsvConverter.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
```
Comment (pauloarantesmachado) [inline web-server/src/main/java/com/domrock/configurator/utils/CsvConverter.java:11]: remove empty line

File: web-server/people.csv
Comment (ChavesVini) [reply web-server/people.csv:1]: Not necessary

File: web-server/people.xlsx
Comment (ChavesVini) [reply web-server/people.xlsx:1]: Not necessary

**Comment by MarcyLeite** [comment]
Follwing what we talk in class, this PR will be closed and not committed. Changes related to this feature will be continued from commit 5891c087cd033f59cf9f436f02d8b7e6131a9bd0 (#9)
